vue/components/ui/modal-platforme/modal-platforme.vue

```diff
@@ -0,0 +1,289 @@
+    opacity: 1;
+    position: fixed;
+    right: 0px;
+    text-align: center;
+    top: 0px;
+    transition: opacity 0.125s ease-in-out;
+    z-index: 10;
+}
+
+.modal.fade-enter,
+.modal.fade-leave-to {
+    opacity: 0;
+}
+
+.modal.fade-leave-active {
+    transition-delay: 0.25s;
+}
+
+.modal.fade-enter-to {
+    opacity: 1;
+}
+
+.modal > .modal-overlay {
+    background-color: rgba(0, 0, 0, 0.5);
+    height: 100%;
+    left: 0px;
+    position: absolute;
+    top: 0px;
+    width: 100%;
+}
+
+.modal > .modal-container {
+    background-color: #ffffff;
+    border-radius: 4px 4px 4px 4px;
+    box-shadow: 0px 0px 12px #2d2d2d;
+    display: inline-block;
+    max-height: 80%;
+    opacity: 1;
+    overflow-y: auto;
+    padding: 40px 40px 40px 40px;
+    position: relative;
+    top: 50%;
+    transform: translateY(-50%);
+    transition: opacity 0.125s ease-out 0.125s, transform 0.25s ease-in-out 0.125s;
+}
+
+body.tablet .modal > .modal-container,
+body.mobile .modal > .modal-container {
+    box-sizing: border-box;
+    max-height: 90%;
+    max-width: 100%;
+    padding: 20px 10px 20px 10px;
+}
+
+.modal.fade-enter > .modal-container,
+.modal.fade-leave-to > .modal-container {
+    opacity: 0;
+    transform: translateY(-48%);
+}
+
+.modal.fade-enter-to > .modal-container {
+    opacity: 1;
+    transform: translateY(-50%);
+}
+
+.modal > .modal-container > .button.button-close {
+    left: 0px;
+    margin: auto;
+    padding: 5px 5px 5px 5px;
+    position: absolute;
+    text-align: left;
+    top: 0px;
+}
+
+.modal > .modal-container > .button.button-close > img {
+    height: 25px;
+    margin: auto;
+    vertical-align: middle;
+    width: 25px;
+}
+
+body.mobile .modal > .modal-container > .button.button-close > img {
+    height: 15px;
+    width: 15px;
+}
+
+.modal > .modal-container > .modal-content {
+    opacity: 1;
+    transition: opacity 0.25s ease-in-out 0.125s;
+}
+
+.modal.fade-enter > .modal-container > .modal-content,
+.modal.fade-leave-to > .modal-container > .modal-content {
+    opacity: 0;
+}
+
+.modal.fade-enter-to > .modal-container > .modal-content {
+    opacity: 1;
+}
+
+.modal > .modal-container > .buttons-container > .button {
+    display: inline-block;
+}
+
+.modal > .modal-container > .buttons-container > .button.button-cancel {
+    margin-right: 20px;
+}
+</style>
+
+<script>
+export const ModalPlatforme = {
+    name: "modal-platforme",
+    props: {
+        applyText: {
+            type: String,
+            default: null
+        },
+        buttonClose: {
+            type: Boolean,
+            default: true
+        },
+        cancelText: {
+            type: String,
+            default: null
+        },
+        globalEvents: {
+            type: Boolean,
+            default: true
+        },
+        name: {
+            type: String,
+            mandatory: true
+        },
+        overlay: {
+            type: Boolean,
+            default: true
+        },
+        overlayLeave: {

```

Comment:
??

Reply:
overlayLeave = true => clicking on the overlay closes the modal